tools/project_manager/projectmgr.php

```diff
@@ -50,8 +50,25 @@
 // exits if handled
 handle_set_cols($show_view, "PM");
 
+$server_timezone = date_default_timezone_get();
+$lang_code = short_lang_code();
+$js_files = [
+    "$code_url/tools/dropdown.js",
+    "$code_url/scripts/time.js",
+    "https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.js",
+    "https://cdnjs.cloudflare.com/ajax/libs/moment-timezone/0.5.23/moment-timezone-with-data.js"
+];
+
+if($lang_code != "en")
```

Comment:
I assume(?) that if there isn't a locale file available that moment will fall back to English, which is great. That at least helps buffer the case where a DP language will be added that moment doesn't support.